packages/playground/src/App.vue

```diff
@@ -317,6 +317,12 @@ const routes: AppRoute[] = [
         route: DashboardRoutes.Deploy.Applications,
         tooltip: "Deploy ready applications on the ThreeFold grid.",
       },
+      {
+        title: "Domains",
```

Comment:
should it be `Domains` or `Domain` ?

Reply:
According to a conversation with @AhmedHanafy725 in the past the solution name was to be changed from ```Expose``` to ```Domains```